scipy/linalg/basic.py

```diff
@@ -85,11 +88,14 @@ def solve(a, b, sym_pos=False, lower=False, overwrite_a=False,
         Whether to check that the input matrices contain only finite numbers.
         Disabling may give a performance gain, but may result in problems
         (crashes, non-termination) if the inputs do contain infinities or NaNs.
+        Default is True.
     assume_a : str, optional
         Valid entries are explained above.
-    transposed: bool, optional
-        If True, ``a^T x = b`` for real matrices, raises `NotImplementedError`
-        for complex matrices (only for True).
+        Default is ``'gen'``.
+    transposed : bool, optional
+        If True, calculate ``a^T x = b``.
```

Comment:
Consider:
```suggestion
        If True, solve ``a.T @ x = b``.
```

Reply:
I would leave something in there about raising an error for complex matrices when it's True.

Reply:
This is something I'm planning to solve with the new version of solve. Then it would be possible to do it with reals and doubles hopefully with a better API.